userguide/plugins.rst

```diff
@@ -142,10 +142,10 @@ plugins are started when the system boots. Unsetting :guilabel:`Boot`
 means the plugin will not start when the system boots and must be
 started manually.
 
-In addition to the :guilabel:`Jail` name, the :guilabel:`Columns` menu controls
-which information is displayed for each installed Plugins entry. Options include
-status, IPv4 address, IPv6 address, installed version, plugin name, release, boot status,
-and collection.
+In addition to the :guilabel:`Jail` name, the :guilabel:`Columns`
+menu controls which information is displayed for each installed
+Plugins entry. Options include status, IPv4 address, IPv6 address,
```

Comment:
Don't list the options that can be enabled, just mention that additional columns can be enabled or disabled with the Columns menu.  We should make this a text replacement, like |columns-desc|.